jsapp/js/account/usage/usageProjectBreakdown.tsx

```diff
@@ -69,90 +34,63 @@ const ProjectBreakdown = () => {
         };
       });
 
-      setProjectData((prevData) => {return {...prevData, results: updatedResults};});
+      if (isComponentMounted) {
```

Comment:
What does `isComponentMounted` do here? If I comment all references to it out, everything seems to function the same, but I could easily be missing something.

Reply:
It was previously used when I had the double fetch call before I had this endpoint. So yes thank you! It should be removed.